src/components/inputs/Switch/Switch.tsx

```diff
@@ -34,14 +33,14 @@ const Switch = (props: IProps) => {
 		style
 	} = props;
 
-	const [isChecked, setIsChecked] = React.useState(checked)
+	const [checked, setChecked] = React.useState(props.checked)
 
-	React.useEffect(() => setIsChecked(checked), [checked])
+	React.useEffect(() => setChecked(props.checked), [props.checked])
 
 	const handleChange = () => {
-		setIsChecked(prev => !prev);
+		setChecked(!checked);
 		if (onChange) {
-			onChange(name, checked);
+			onChange(name, !checked);
```

Comment:
Good catch, sorry about that! Hope that wasn't a huge headache but I bet it was

Reply:
Lol, no worries. I just thought I was going crazy for a minute when things stopped working in production 😆